### PR TITLE
build: minimum version 24

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Set android 7 as minimum version for miniapp sdk.
Using the latest function `shouldOverrideUrlLoading`, no need to use deprecated function supported android 6 anymore.

## Links
MINI-2648

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: build.gradle [hunk 1]
```diff
@@ -35,7 +35,7 @@ buildscript {
 
 //  Build Config.
     apply from: 'config/index.gradle'
-    CONFIG.versions.android.sdk.min = 23
+    CONFIG.versions.android.sdk.min = 24
```
[the remark this answers is not in the document]
Comment (minh-rakuten) [reply build.gradle:38]: Thanks, updated.

**Review by corycaywood — COMMENTED** [review]
Looks good! Just need to add this to the changelog too.

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**